subprojects/packagefiles/libuv/link_file_in_build_dir.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 
-import os, sys
+import os
+import sys
```

Comment:
Ugh, please can we not!

Reply:
Can we follow standard python coding guidelines? yes

Reply:
Nothing standard about this change, other than that black encourages it.

Reply:
This actually comes from pep8.

https://peps.python.org/pep-0008/#imports

Reply:
The authors and maintainers of the cpython PEPs take every possible opportunity they possibly can to tell people that PEP 8 is ***not*** a standard for anything other than "the style for internal cpython project source code" and that people should stop putting them in a position to be the arbiters of python style, funnily enough.

And this particular rule is inconsistent and ugly so I can kind of see the point. :)